src/scripts/components/dd-browser-logs-rum.js

```diff
@@ -28,7 +28,10 @@ datadogLogs.init({
 if (env === 'preview' || env === 'live') {
     datadogRum.init({
         applicationId: ddApplicationId,
-        clientToken: ddClientToken
+        clientToken: ddClientToken,
+        env: env,
```

Comment:
you can do: 
```suggestion
        env,
```
to prevent the eslint warning